opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml

```diff
@@ -1036,3 +1038,27 @@ components:
           format: date
         requestedExecutionTime:
           type: string
+
+    ChallengeData:
+      title: SinglePayment
```

Comment:
Is it single payment? Probably you meant ChallengeData

Reply:
Sorry, yes. Renamed